conf/messages

```diff
@@ -317,13 +317,14 @@ viewAllMovements.sort.newest = Dispatched (newest)
 viewAllMovements.sort.oldest = Dispatched (oldest)
 
 viewAllMessages.title = Messages
-viewAllMessages.heading = Messages({0})
+viewAllMessages.heading = Messages ({0})
 viewAllMessages.totalNumberOfMessages = You have {0} unread messages.
 viewAllMessages.sort.label = Sort by
+viewAllMessages.sort.button = Sort messages
 viewAllMessages.sort.messageTypeA = Message (A - Z)
 viewAllMessages.sort.messageTypeD = Message (Z - A)
-viewAllMessages.sort.dateReceivedA = Received (newest)
-viewAllMessages.sort.dateReceivedD = Received (oldest)
+viewAllMessages.sort.dateReceivedD = Received (newest)
+viewAllMessages.sort.dateReceivedA = Received (oldest)
 viewAllMessages.sort.arcA = Arc (A - Z)
 viewAllMessages.sort.arcD = Arc (Z - A)
```

Comment:
Unrelated to what you've done, but should Arc be captialised?

Reply:
Sorting by ARC doesn't seem to be in the acceptance criteria?

Reply:
> Sorting by ARC doesn't seem to be in the acceptance criteria?

The **messages-FrontendAPISpecification-MDTPtoEIS** document has only 4 possible sort values [messagetype, datereceived, arc & readindicator]

Reply:
I think that was the intention, probably worth checking with UCD?